netlify.toml

```diff
@@ -1,3 +1,6 @@
+[build]
+  command = "yarn build:website"
+
 [build.environment]
-  NODE_VERSION = "14"
+  NODE_VERSION = "16"
```

Comment:
Just a question, what feature of Node.js 16 we are using?

Reply:
Also, I think we should mention in CONTRIBUTING.md that we need Node.js 16 for building our project.

Reply:
It's not required, just want to use the newer version.

Reply:
I prefer using Node.js 14 that is current LTS version.